packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -3098,4 +3098,117 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("PushInt", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should push uint64 to stack", () => {
+      const op = new PushInt([MAX_UINT64.toString()], 0);
+      op.execute(stack);
+
+      assert.equal(1, stack.length());
+      assert.equal(MAX_UINT64, stack.pop());
+    });
+  });
+
+  describe("PushBytes", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should push bytes to stack", () => {
+      const str = "\"Algorand\"";
+      const bytes = new Uint8Array(Buffer.from("Algorand"));
```

Comment:
```suggestion
      const expected_bytes = new Uint8Array(Buffer.from("Algorand"));
```

Reply:
I think it should be `expectedBytes`, not `expected_bytes`